public/v4/apps/languagetool.yml

```diff
@@ -0,0 +1,50 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: erikvl87/languagetool:$$cap_version
+        restart: always
+        ports:
+            - $$cap_port:8010
+        environment:
+            langtool_languageModel: /ngrams # OPTIONAL: Using ngrams data
+            Java_Xms: $$cap_min_size # OPTIONAL: Setting a minimal Java heap size of 512 mib
+            Java_Xmx: $$cap_max_size # OPTIONAL: Setting a maximum Java heap size of 1 Gib
+            #langtool_pipelinePrewarming: true
+        volumes:
+            - $$cap_appname-data:/ngrams
+        caproverExtra:
+            notExposeAsWebApp: 'true'
```

Comment:
Why is this not a web app? I'm not too familiar with the tool but from the looks of it, I think it's an HTTP app. No?

Reply:
No it is not

For exemple if the Firefox extension need to check `my text`, a request will be send in this format:
http://$$cap_appname.$$cap_root_domain:$$cap_port/v2/check?language=en-US&text=**my+text**
The server will reply with a json see https://languagetool.org/http-api/swagger-ui/#/default and this https://dev.languagetool.org/http-server.html#testing-the-server

Reply:
```
http://$$cap_appname.$$cap_root_domain:$$cap_port/v2/check?language=en-US&text=my+text
```
This is an http request, hence a web app, it can be routed through nginx, no need for port mapping.

Reply:
"web app" doesn't necessary mean it has a GUI - it just means that it can be routed through nginx as it uses http(s).

Reply:
I don't understand, it didn't work last time, I must mess with something last time
I learn something new today

Thank you for making CapRover so good